src/Sylius/Bundle/ReviewBundle/Resources/translations/validators.en.yml

```diff
@@ -0,0 +1,10 @@
+sylius:
+    review:
+        title:
+            not_blank: Review title should not be blank.
+            min_length: Review title must have at least 2 characters
+            max_length: Review title must have at most 255 characters
+        rating:
+            not_blank: You must check review rating.
+        author:
+            already_exists: This email is already registered. Please log in.
```

Comment:
Review bundle should have no clue about users. It cares only about review authors, but definitely not about whether they are authenticated or not. I know it is only a translation, but I fear we may have a bigger coupling issue in the logic itself.